nix/nixpkgs.nix

```diff
@@ -46,12 +46,34 @@ let
       ];
     });
     scala_2_12 = pkgs.scala_2_12.overrideAttrs (oldAttrs: rec {
+      version = "2.12.12";
       name = "scala-2.12.12";
       src = pkgs.fetchurl {
         url = "https://www.scala-lang.org/files/archive/${name}.tgz";
         sha256 = "3520cd1f3c9efff62baee75f32e52d1e5dc120be2ccf340649e470e48f527e2b";
       };
     });
+   haskell = pkgs.haskell // {
+     packages = pkgs.haskell.packages // {
+       integer-simple = pkgs.haskell.packages.integer-simple // {
+        ghc8103 = pkgs.haskell.packages.integer-simple.ghc8103.override {
+          ghc = pkgs.haskell.compiler.integer-simple.ghc8103.overrideAttrs (old: {
```

Comment:
Why is this needed?

Reply:
Otherwise GHC fails to find `otool`. Added a comment (I already had, just forgot to push it :facepalm:)